File: java/arquivos/EscreverArquivo.java
/**
 * Programa Java usa o componente Arquivo para escrever (adicionar) conteúdo no arquivo teste.txt
 * O arquivo manipulado por esse programa fica contido no diretório corrente.
 *
 * Objetivos:
 *   - escrever conteúdo no arquivo da pasta corrente.
 *
 * Módulo: Lógica de programação | Capítulo: XXX | Página: XXX
 * Author: Eder Magalhães
 */

public class EscreverArquivo {

    public static void main(String[] args) {
        String nomeArquivo = "teste.txt";
        Arquivo meuArquivo = Arquivo.abrirEscrita(nomeArquivo, "append");
        
        Arquivo.escrever(meuArquivo, "Olá, este é meu primeiro arquivo");
        Arquivo.escrever(meuArquivo, "Testando gravação de dados em arquivo texto (em Java).");

        Arquivo.fechar(meuArquivo);

        System.out.println("Conteúdo foi armazenado no arquivo " + nomeArquivo + "!");
    }

}

File: python/arquivos/escreverArquivo.py
# -*- coding: utf-8 -*-

# Programa Python demonstra como escrever (adicionar) conteúdo no arquivo teste.txt.
# Objetivos:
#   - escrever conteúdo no arquivo da pasta corrente.
#
# Módulo: Lógica de programação
# Author: Eder Magalhães

nomeArquivo = "teste.txt"

meuArquivo = open(nomeArquivo, 'a');
meuArquivo.write("Olá, este é meu primeiro arquivo.\n");
meuArquivo.write("Testando gravação de dados em arquivo texto (em Python).\n");

meuArquivo.close();

print "Conteúdo foi armazenado no arquivo ", nomeArquivo, "!";

